docs/tagging.md

```diff
@@ -0,0 +1,64 @@
+|---------------------------------|---------------------------|--------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CSIVolumeName                   | pvcName                   | CSIVolumeName = pvc-a3ab0567-3a48-4608-8cb6-4e3b1485c808                 | Added to all volumes.                                                                                                                                                                                                                             |
+| CSIVolumeSnapshotName           | volumeSnapshotContentName | CSIVolumeSnapshotName = snapcontent-69477690-803b-4d3e-a61a-03c7b2592a76 | Added to all snapshots.                                                                                                                                                                                                                           |
+| fsx.openzfs.csi.aws.com/cluster | true                      | fsx.openzfs.csi.aws.com/cluster = true                                   | Add to all volumes, snapshots, and final backups. This allows users to track which resources were created via the FSx for OpenZFS CSI driver. Users may also use a policy to limit the csi driver's permissions to just the resources it manages. |                                                                                      |
+
+# StorageClass Tagging
+
+The AWS FSx for OpenZFS CSI Driver supports tagging through `StorageClass.parameters`
+
+**Example 1**
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fsx-openzfs-sc
+provisioner: fsx.openzfs.csi.aws.com
+parameters:
+  ...
+  ...
+  tags: "key1=value1,key2=hello world,key3="
+```
+
+Provisioning a volume using this StorageClass will apply the following tags:
+
+```
+key1=value1
+key2=hello world
+key3=<empty string>
+```
+
+# Snapshot Tagging
+The AWS FSx for OpenZFS CSI Driver supports tagging snapshots through `VolumeSnapshotClass.parameters`, similar to StorageClass tagging.
+
+**Example**
+```
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-aws-vsc
+driver: fsx.openzfs.csi.aws.com
+deletionPolicy: Delete
+parameters:
+  tags: "key1=value1,key2=hello world,key3="
+```
+
+Provisioning a snapshot in namespace 'ns-prod' with `VolumeSnapshot` name being 'fsx-openzfs-snapshot' using this VolumeSnapshotClass, will apply the following tags to the snapshot:
```

Comment:
Not sure if namespace and snapshot name is relevant to what tags are added based on user input